core/wiki/macros/toc_template_external-link.tid

```diff
@@ -0,0 +1,7 @@
+title: $:/core/macros/toc/template/external-link
+type: text/vnd.tiddlywiki
+
+<$navigator story="$:/StoryList" history="$:/HistoryList" openLinkFromInsideRiver={{$:/config/Navigation/openLinkFromInsideRiver}} openLinkFromOutsideRiver={{$:/config/Navigation/openLinkFromOutsideRiver}} relinkOnRename={{$:/config/RelinkOnRename}}>
```

Comment:
The navigator widget is quite heavy weight, do we really have to have an instance for each external link in the TOC?

Reply:
Yea, I know. .. I'll have a look if it can be done with `tv-` variables only. .. Or may be linkcatcher.